packages/analytics-js-integrations/src/integrations/Ninetailed/browser.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import {
+  NAME,
+  DISPLAY_NAME,
+} from '@rudderstack/analytics-js-common/constants/integrations/Ninetailed/constants';
+import Logger from '../../utils/logger';
+
+const logger = new Logger(DISPLAY_NAME);
+class Ninetailed {
+  constructor(config, analytics, destinationInfo) {
+    if (analytics.logLevel) {
+      logger.setLogLevel(analytics.logLevel);
+    }
+    this.analytics = analytics;
+    this.organisationId = config.organisationId;
+    this.environment = config.environment;
+    this.name = NAME;
+    ({
+      shouldApplyDeviceModeTransformation: this.shouldApplyDeviceModeTransformation,
+      propagateEventsUntransformedOnError: this.propagateEventsUntransformedOnError,
+      destinationId: this.destinationId,
+    } = destinationInfo ?? {});
+  }
+
+  init() {
+    // We expect the customer to load the SDK through NPM package
+    // Docs: https://docs.ninetailed.io/for-developers/experience-sdk/getting-started
+  }
+
+  isLoaded() {
+    return !!window.ninetailed;
+  }
+
+  isReady() {
+    return this.isLoaded();
+  }
+
+  identify(rudderElement) {
+    const { message } = rudderElement;
+    const { userId, traits, context } = message;
```

Comment:
`message.traits` contain the group traits. User traits are only available at `message.context.traits`.
What are we interested in?

Reply:
For identify events, group traits (`message.traits`) are not available but user traits (`message.context.traits`) are available.

For group events, group traits and user traits are available.

Reply:
Yes.

Reply:
Since group call is not supported in ninetailed message.traits does not make sense hence removing it